client/state/admin-menu/fallback-data.json

```diff
@@ -0,0 +1,479 @@
+[
+    {
+        "icon": "dashicons-feedback",
+        "slug": "feedback",
+        "title": "Feedback",
+        "type": "menu-item",
+        "url": "https://wordpress.com/wp-admin/?page=feedback",
+        "children": {
+            "1": {
+                "parent": "feedback",
+                "slug": "edit-phppost_typefeedback",
+                "title": "Feedback",
+                "type": "submenu-item",
+                "url": "https://wordpress.com/wp-admin/edit.php?post_type=feedback"
+            },
+            "2": {
+                "parent": "feedback",
+                "slug": "polls",
+                "title": "Polls",
+                "type": "submenu-item",
+                "url": "https://wordpress.com/wp-admin/admin.php?page=polls"
+            },
+            "3": {
+                "parent": "feedback",
+                "slug": "ratings",
+                "title": "Ratings",
+                "type": "submenu-item",
+                "url": "https://wordpress.com/wp-admin/admin.php?page=ratings"
+            }
+        }
+    },
```

Comment:
This menu item should display after Comments. See D49299-code for a fix for that on the wpcom side

Reply:
I think this is the first example of us needing to update the static data to reflect the API endpoint. I do worry this task will be ongoing until / if we launch the feature and then afterward.

Reply:
It shouldn't be, with the spreadsheet we have a predefined order that we can use.